src/components/Calculrator.tsx

```diff
@@ -0,0 +1,29 @@
+import useCalculate from "@/hook/useCalculate";
+import CalculateDisplay from "./CalculateDisplay";
+import Digit from "./Digit";
+import Modifier from "./Modifier";
+import Operation from "./Operation";
+
+const Calculrator = () => {
+  const {
+    handleDigit,
+    handleOperation,
+    handleCalculate,
+    reset,
+    calculrateState,
+  } = useCalculate();
+
+  return (
+    <div className="calculator">
+      <CalculateDisplay calculrateState={calculrateState} />
```

Comment:
```suggestion
      <CalculateDisplay {...calculrateState} />
```
이렇게 펼쳐서 내려주면 어떨까 싶네요!